.kokoro/build.sh

```diff
@@ -112,6 +112,25 @@ clirr)
     mvn -B -ntp -Denforcer.skip=true clirr:check
     RETURN_CODE=$?
     ;;
+conformance)
+    # Build and start the proxy in a separate process
+    pushd .
+    cd test-proxy
+    mvn clean install
```

Comment:
I think we can skip the other tests here: 
```suggestion
    mvn clean install -DskipTests
```

Reply:
Done